src/Mutation/UserRegister.php

```diff
@@ -133,11 +133,21 @@ public static function mutate_and_get_payload() {
 			 */
 			unset( $user_args['role'] );
 
+			/**
+			 * Disable "Password Changed" emails from being sent.
+			 */
+			add_filter( 'send_password_change_email', '__return_false' );
+
 			/**
 			 * Update the registered user with the additional input (firstName, lastName, etc) from the mutation
 			 */
 			wp_update_user( $user_args );
 
+			/**
+			 * Re-enable "Password Changed" emails.
+			 */
+			remove_filter( 'send_password_change_email', '__return_false' );
```

Comment:
@kellenmace I think your point in the issue about removing the filter is valid. 

What if we do something like this: 

```
// Apply the filter to get the current filtered value
$filter_value_before_update = apply_filters( 'send_password_change_email', '__return_false' );

// Add filter to make it explicitly false
add_filter( 'send_password_change_email', '__return_false' );

// Update the user, which will call the function with the filter
wp_update_user( $user_args );

// Reset the filter to the value it was before this function
add_filter( 'send_password_change_email', $filter_value_before_update );
```